### ISSUE TITLE
Documentation Accessibility - Color Scheme

### ISSUE THREAD

**Issue by JoeCalvert** [VoronDesign/Voron-Documentation#316]
### What's the problem?

The color scheme of the docs site gives titles a very low contrast ratio. Being colourblind, this makes them hard to read.
Updating the color scheme to follow WCAG 2 guidelines would provide a much better experience for general users and improve readability for users that have issues with their vision

[WAVE report](https://wave.webaim.org/report#/https://docs.vorondesign.com/hardware.html)
[WhoCanUse Contrast Assessment](https://www.whocanuse.com/?bg=2c2f33&fg=cc000e&fs=20&fw=)

### Fixes

- [ ] Decide on a color palette that can be used to update the design of the documentation site
- [ ] Assess the color palette for accessibility against WCAG 2 guidelines
- [ ] Update the site theme with the new colours

### Suggestions

Updating the theme provides an opportunity to update the design tokens found in [voron-dark.scss](https://github.com/VoronDesign/Voron-Documentation/blob/33b322f7d817b601c270b7cdab878ad047a46421/_sass/color_schemes/voron-dark.scss) to be self-documenting. For example, having tokens such as `background-dark` and `on-background-dark` instead of `grey-dk-400` and `voron-red-60`. This could help to reduce design issues in future changes

**Comment by nemgrea** [comment]
i believe this should be resolved by pull request https://github.com/VoronDesign/Voron-Documentation/pull/293
